pkg/descheduler/controllers/migration/arbitrator/arbitrator.go

```diff
@@ -102,13 +111,38 @@ func (a *arbitratorImpl) Add(job *v1alpha1.PodMigrationJob) {
 	a.waitingCollection[job.UID] = job.DeepCopy()
 }
 
+// Delete remove PodMigrationJob from local PassedArbitration Record map.
+func (a *arbitratorImpl) Delete(uid types.UID) {
```

Comment:
As we talked, `Add/Delete` is not better, recommend to `AddPodMigrationJob/DeleteMigrationJob`

Reply:
yes